tests/Metadata/RootMetadataTest.php

```diff
@@ -40,7 +40,6 @@ public function providerExpectedField(): array
 
         $data[] = ['signed:keys'];
         $firstKey = $this->getFixtureNestedArrayFirstKey('1.root.json', ['signed', 'keys']);
-        $data[] = ["signed:keys:$firstKey:keyid_hash_algorithms"];
```

Comment:
Why was this removed?

Reply:
I realized the `\Tuf\Metadata\ConstraintsTrait::getKeyConstraints()` we had the validator for set up wrong. Check out the change there. Basically it should have been optional but it wasn't. It will be removed at some point in the Python library, see #174 and #179.

So this provider was checking that this field was *not* optional. It only passed because `getKeyConstraints()` was wrong. See `\Tuf\Tests\Unit\KeyTest::providerCreateFromMetadata()` that should we can validate now with or without this field.